src/containers/Article/Read/index.js

```diff
@@ -159,18 +160,19 @@ class Read extends Component {
                 </div>
 
                 <div className="col s1">
-                  <div className="reactions">
+                  <div className="reaction">
                     {this.renderReaction('like', thumbsUp, payload.article.likes_count)}
                     {this.renderReaction('dislike', thumbsDown, payload.article.dislikes_count)}
```

Comment:
Let's destructure payload.article here

Reply:
Okay

Reply:
I have trie to destructure but it throws an error because we are trying to destructure from a payload that has not resolved yet.

Reply:
Since we are hours away from the Demo, let's ignore this for now